From febf81612a5e57fc6f8144c45fbd21550cb4f8fb Mon Sep 17 00:00:00 2001
From: Yuwei Chen <yuwei.chen@intel.com>
Date: Mon, 3 Aug 2020 10:04:32 +0800
Subject: [PATCH] Test case5 for Change value of FeatureFlag PCD

Description: Change the FeatureFlag PCD value to ("gUefiOvmfPkgTokenSpaceGuid.PcdSmmSmramRequire|FALSE") in TestPkg/TestPkg32X64.dsc.

Repo: edk2
Signed-off-by: Yuwei Chen <yuwei.chen@intel.com>
---
 TestPkg/TestPkg32X64.dsc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/TestPkg/TestPkg32X64.dsc b/TestPkg/TestPkg32X64.dsc
index 532d084223..dbf940f989 100644
--- a/TestPkg/TestPkg32X64.dsc
+++ b/TestPkg/TestPkg32X64.dsc
@@ -176,7 +176,7 @@
 #
 ################################################################################
 [PcdsFeatureFlag]
-gUefiOvmfPkgTokenSpaceGuid.PcdSmmSmramRequire|TRUE
+gUefiOvmfPkgTokenSpaceGuid.PcdSmmSmramRequire|FALSE
 gEfiMdeModulePkgTokenSpaceGuid.PcdPeiFullPcdDatabaseEnable|TRUE
 
 
-- 
2.27.0.windows.1

